charts/prometheus/values.yaml

```diff
@@ -48,3 +48,36 @@ ports:
   http: 9090
 
 ingressNetworkPolicyExtraSelectors: []
+
+
+astroHTTPTargets:
+  commander: false
```

Comment:
@jpweber why disable commander target by default?

Reply:
This probably should default to true here as well. This might be left over from me testing things